internal/controllers/nmc_reconciler_test.go

```diff
@@ -205,6 +229,8 @@ var _ = Describe("NodeModulesConfigReconciler_Reconcile", func() {
 		contextWithValueMatch := gomock.AssignableToTypeOf(
 			reflect.TypeOf((*context.Context)(nil)).Elem(),
 		)
+		kubeClient.EXPECT().Get(ctx, types.NamespacedName{Name: nmc.Name}, &node).Return(err)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
Same as https://github.com/kubernetes-sigs/kernel-module-management/pull/850/files#r1677111464

Reply:
fixed, moved `kubeClient.EXPECT().Get(ctx, types.NamespacedName{Name: nmc.Name}, &node).Return(err)` to the InOrder section (`Expect(err).NotTo(HaveOccurred())` should not be there indeed)